version/docs/version.asciidoc

```diff
@@ -1,4 +1,4 @@
-:stack-version: 8.13.1
+:stack-version: 8.13.2
```

Comment:
I'm not 100% sure we even use this, possibly we can remove this automated PR. There are no docs in this repository and no docs build.

Reply:
I will take a look for next releases.